packages/agreement-process/src/model/domain/toEvent.ts

```diff
@@ -112,3 +114,21 @@ export function toCreateEventAgreementUpdated(
     },
   };
 }
+
+export function toCreateEventAgreementContractAdded(
+  agreementId: string,
+  agreementDocument: AgreementDocument,
+  version: number
```

Comment:
In the last commit those updates require `version` as parameter, it's done to guarantee the properly sequential updates.